source/agora/common/crypto/Key.d

```diff
@@ -271,7 +296,7 @@ public struct SecretKey
 
     ***************************************************************************/
 
-    public Signature sign (scope const(ubyte)[] msg) const
+    public Signature sign (scope const(ubyte)[] msg) const nothrow @nogc
```

Comment:
This should ideally be moved into its own commit. So one commit replacing the `nothrow @nogc:` with this, and a separate commit which only implements what the issue is about.